client/src/app/site/mediafiles/models/view-file.ts

```diff
@@ -0,0 +1,59 @@
+import { BaseViewModel } from '../../base/base-view-model';
+import { Mediafile } from '../../../shared/models/mediafiles/mediafile';
+import { User } from '../../../shared/models/users/user';
+import { BaseModel } from '../../../shared/models/base/base-model';
+
+export class ViewFile extends BaseViewModel {
```

Comment:
Please use Mediafile.

Reply:
Renamed ViewFile in ViewMediafile + changed the filename